dockers/base-cuda/Dockerfile

```
@@ -114,6 +115,10 @@ RUN \
     # install DeepSpeed
     pip install deepspeed==0.5.4
 
+RUN \
+    # install Bagua
+    pip install bagua-cuda${BAGUA_CUDA_VERSION}==0.9.0
```

I think we can get rid of `BAGUA_CUDA_VERSION` above, and do the following:
```suggestion
    pip install bagua-cuda${CUDA_VERSION//.}==0.9.0
```

---

It just removes `.` in the variable:
```console
$ export SOME_VAR="10.2"
$ echo ${SOME_VAR//.}
102
```

---

Thanks  @akihironitta